src/base/LruMap.h

```diff
@@ -53,9 +66,17 @@ template <class EntryValue, size_t EntryCost = sizeof(EntryValue)> class LruMap
     /// The free space of the map
     size_t freeMem() const { return (memLimit() > size() ? memLimit() - size() : 0);}
     /// The current size of the map
-    size_t size() const {return (entries_ * EntryCost);}
+    size_t size() const {return memUsed_;}
```

Comment:
To avoid misunderstanding, I consider this particular issue addressed. Thank you.